src/init.lua

```diff
@@ -1,20 +1,26 @@
-local RunService = game:GetService("RunService")
-local ServerScriptService = game:GetService("ServerScriptService")
 local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local ServerScriptService = game:GetService("ServerScriptService")
+local RunService = game:GetService("RunService")
 
-local ServerFolder = script:FindFirstChild("server") --server will already have moved this on client join
-local SharedFolder = script:WaitForChild("shared")
+local ServerFolder
+local SharedFolder
 
-local ReplicaService = ServerFolder and ServerFolder:WaitForChild("ReplicaService") or nil
-local ReplicaController = SharedFolder:WaitForChild("ReplicaController")
+local ReplicaService
+local ReplicaController
 
-if RunService:IsServer() then
-	ServerFolder.Parent = ServerScriptService
-else
-	SharedFolder.Parent = ReplicatedStorage
+if (RunService:IsServer()) then
+  ServerFolder = script.Parent:WaitForChild("Server")
+  ServerFolder.Name = "_RS_Server"
+  ServerFolder.Parent = ServerScriptService
+  ReplicaService = ServerFolder:WaitForChild("ReplicaService")
+elseif (RunService:IsClient()) then
+  SharedFolder = script.Parent:WaitForChild("Shared")
+  SharedFolder.Name = "_RS_Shared"
+  SharedFolder.Parent = ReplicatedStorage
+  ReplicaController = SharedFolder:WaitForChild("ReplicaController")
 end
 
 return {
-	ReplicaService = RunService:IsServer() and require(ReplicaService) or {},
-	ReplicaController = RunService:IsClient() and require(ReplicaController) or {},
+  ReplicaService = (RunService:IsServer() and ReplicaService) and require(ReplicaService) or {},
+  ReplicaController = (RunService:IsClient() and ReplicaController) and require(ReplicaController) or {}
 }
```

Comment:
I don't think these changes are necessary, the old loader worked fine and this *might* be a breaking change for some users of the package.